client/log_client_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/proto" //nolint:staticcheck
```

Comment:
Is the nolint temporary or related to some other thing we need to address or something else?

Reply:
The nolint will be there until we do a full migration to proto v2. 

See full explanation here:
https://blog.golang.org/protobuf-apiv2

Reply:
Thanks.